app.py

```diff
@@ -59,6 +65,93 @@ def updated():
         return redirect(url_for('index'))
 
 
+### Module 2 stuff ### 
+
+@app.route('/items/get_all_cards', methods = ["GET"])
+def getAll(): 
+    resp = trelloapp.get_all_cards_from_todo_list()
+    resp_list = trelloapp.get_all_cards_from_done_list()
+    #print(resp)
+    resp_json = json.loads(resp)
+    resp_json_list = json.loads(resp_list)
+    for cards in resp_json:   
+        print(cards['name'])
+    for lists in resp_json_list:   
+        print(lists['id'] + lists['name'])
+    return render_template('all_items.html', cards_from_todo = resp_json, cards_from_done = resp_json_list)
+
+
+@app.route('/complete_item', methods = ['POST', 'GET'])
+def complete_item():
+    if request.method == 'POST':
+        card_name = request.form['card_name']
+        card_id = request.form['card_id']
+        
+        trelloapp.move_card_to_done(card_id)
+    return "Card moved to Done, Please check Trello!"
+
+@app.route('/back_todo_item', methods = ['POST', 'GET'])
+def back_todo_item():
+    if request.method == 'POST':
+        card_name = request.form['card_name']
+        card_id = request.form['card_id']
+        
+        trelloapp.move_card_to_do(card_id)
+    return "Card moved to To Do, Please check Trello!"
+
+# return the list ID from resp_json_list for name "Done"
+# call update function from trellop to update the listID for cardID
+
+
+
+# list ID, card ID, card Name
+
+#getAll()
+
+#pass the list name and card name into complete_item
+#@app.route('/items/complete_item', methods = ["GET"])
+#def update_to_done():
+#    resp = trellopapp.
+
+
+
+# #Module2 project #
+# APP_KEY = os.environ.get('APP_KEY')
+# APP_TOKEN=os.environ.get('APP_TOKEN')
+# board_id= '83tSYoun'
+# url = 'https://api.trello.com/1/cards/{id}'
+# headers = {"Accept": "application/json"}
+# query = {"key": APP_KEY, "token": APP_TOKEN}
+
+# response=requests.request("GET", url, headers=headers, params=query)
+# print(json.dumps(json.loads(response.text), sort_keys=True, indent=4, separators=(",", ": ")))
+
+#CONTROL K U / CONTROL K C
+# def get_card_from_list(card_id):
+#    parameters = {"key": APP_KEY, "token": APP_TOKEN}
+#    response = request.get(url, % card_id, params=parameters)
+
+#    if response.status_code == 200: 
+#        resp = response.json()
+#        myToDo = []
+#        for result in resp['all']:
+#            myToDo.append({
+#                'CardID': result["idList"],
+#                'CardName': result["name"]
+#            })
+#        return render_template("trello-api.html", myToDo=myToDo)
+#    else: 
+#        return f"There is a problem with the request" 
+
+#@app.route('/ToDo/CreateCard')
```

Comment:
Removing the ability to add todos is a regression from Module 1 and will need to be working again for this to pass review.